libbeat/outputs/fileout/docs/fileout.asciidoc

```diff
@@ -49,6 +49,14 @@ The default value is `true`.
 The path to the directory where the generated files will be saved. This option is
 mandatory.
 
+The path may include the timestamp when the process is started using the `+FORMAT` syntax where `FORMAT` is a
```

Comment:
"process is started" -> file output is initialized

If the outputs can be reloaded dynamically then this different might matter someday.